homeassistant/components/homematicip_cloud/climate.py

```diff
@@ -208,7 +217,7 @@ async def async_set_temperature(self, **kwargs: Any) -> None:
     async def async_set_hvac_mode(self, hvac_mode: HVACMode) -> None:
         """Set new target hvac mode."""
         if hvac_mode not in self.hvac_modes:
-            return
+            raise ValueError(f"Unsupported hvac mode: {hvac_mode}")
```

Comment:
This makes sense, but I think it should be in a different PR?

Reply:
Please move this and the removed check for `preset_mode` to a separate PR about improvin error handling in homematicip_cloud 👍